opentracing-spring-jaeger-starter/src/main/java/io/opentracing/contrib/java/spring/jaeger/starter/JaegerAutoConfiguration.java

```diff
@@ -65,8 +65,10 @@
   public io.opentracing.Tracer tracer(Sampler sampler,
                                       Reporter reporter) {
 
-    final JaegerTracer.Builder builder =
-        new JaegerTracer.Builder(serviceName)
+    // Serive name needs to be a system property to be picked up because of how fromEnv() works.
+    System.setProperty("JAEGER_SERVICE_NAME", serviceName); 
+
+    final JaegerTracer.Builder builder = io.jaegertracing.Configuration.fromEnv().getTracerBuilder()
```

Comment:
I really don't want to do this in an uncontrolled manner. If this is absolutely needed, it should be controlled by a boolean property that is off by default